### PR TITLE
Update to ICANN and private SLDs on CentralNic registry platform

### PR DESCRIPTION
**Pull request by gbxyz**
* [x] Description of Organization
* [x] Reason for PSL Inclusion
* [x] DNS verification via dig
* [ ] Run Syntax Checker (make test)

Description of Organization
====

Organization Website: https://centralnicregistry.com.

CentralNic provides backend registry services to gTLD, ccTLD and SLD registry operators. This pull request adds or updates the entries for a number of domain extensions that we operate on behalf of our clients, and disclaims a number of domain names we no longer operating registry services for.

Reason for PSL Inclusion
====

All the domain extensions referenced in this pull request operate like top-level domain registries: third parties can register domain names via a wide range of registrars, manage their own DNS on thoese domains, and so on.

For some of the domain extensions in this list (e.g. com.fm, edu.vg, and others) this reflects officially mandated policy originating from the sponsoring organisation of the parent TLD - these entries are in the "ICANN" section of the file. Some extensions are operated without official approval from the parent TLD, and so are included in the "private" section.

DNS Verification via dig
=======

```
$ for domain in shop.ro co.ro radio.fm radio.am nz.basketball aus.basketball web.in gov.gd edu.gd org.fm net.fm edu.fm com.fm ; do dig +dnssec @9.9.9.9 +noall +answer TXT _psl.$domain ; echo ; done
_psl.shop.ro.		3600	IN	TXT	"https://github.com/publicsuffix/list/pull/1089"
_psl.shop.ro.		3600	IN	RRSIG	TXT 8 3 3600 20200919064146 20200820074529 351 shop.ro. ah2CDZWKQYNKk6pl4l1qhyXsyBD0zCt5K++3jJF+bihzlf0/SnmcQA7N vJ4O3aD1G9zbxr5ZZWCAuKQO7raPVuOqv4DMIsd0QC6dz0/vGF6AZYBc PP0ygur0hC5n+hJmw0HIRAfo4DsbKQ6fkmFXZnPtUJK6regF8axxgdL+ yYk=

_psl.co.ro.		43200	IN	TXT	"https://github.com/publicsuffix/list/pull/1089"
_psl.co.ro.		86400	IN	RRSIG	TXT 8 3 86400 20200919081910 20200820074538 20100 co.ro. lCF0ju18M/p0LmYSpWXWkiNqc3eQis4rQcS69PJZhtNbVcXf0rxzG1Im /F+tTRecy1129pdb11LzRilWeE8yIlmnyN8d494NMBq2pRR5HSnNNgy2 gg+a0VXDkwSNFu3/sR3hbHJ+k33RVIGW2mMm93Sdl9HwYeIWTARsThj6 uxY=

_psl.radio.fm.		43200	IN	TXT	"https://github.com/publicsuffix/list/pull/1089"
_psl.radio.fm.		43200	IN	RRSIG	TXT 8 3 86400 20200919000406 20200820074546 37981 radio.fm. Xnw/8+9mRt8S4F6b+uXCrEFRXv6+CQworLd2gX2GjGZozELU/SQsyqNQ blgz+acrfOsSQo4JajQH0SGJi2m8ug09u8f43+WTHjCESJCjL3ple2fC afyM/WUHKbgSqzF4QTGOs8CrUDTeGbSgAQS7xYtrqh2/PZxFX7n0jlqh xPk=

_psl.radio.am.		43200	IN	TXT	"https://github.com/publicsuffix/list/pull/1089"
_psl.radio.am.		43200	IN	RRSIG	TXT 8 3 86400 20200919142231 20200820081312 8026 radio.am. aG5R9R1CQjcHzh/v3Cp5z0Q6WeqaCfHpZeYlLvYg12/PtBlvBZDNqdtq STQjW+W6ynyXv9hlEQvA638wqR1F2VS/E7a5hNB2vfhY2QtESlkWnK66 xYU6tSM5r26WbbGlGAubdiVk38nHmx4f2uocHHOA3S3LKGWJtyV1mN73 bqY=

_psl.nz.basketball.	43200	IN	TXT	"https://github.com/publicsuffix/list/pull/1089"
_psl.nz.basketball.	43200	IN	RRSIG	TXT 8 3 86400 20200919022408 20200820081236 5313 nz.basketball. BFCuFggfhXbnSHMJi/pElKUv6p3SoJj1JF30G/ROCzwkz2NmmIPsSz/G Ji2JfM8s2zeDVU+kzeVvFKFfE5ba6pAyAzJlFjZn9+cf2OPcxuFx9xVY Twq+5FIckJo4k6diN6RTXSq8NKZerW2MASyR9F8B9+EMsnZ1G26aLU7c M7g=

_psl.aus.basketball.	43200	IN	TXT	"https://github.com/publicsuffix/list/pull/1089"
_psl.aus.basketball.	43200	IN	RRSIG	TXT 8 3 86400 20200919012707 20200820081237 47115 aus.basketball. fOBSY/XVNQSgXg6tPJKYx4zGFNKZdMOgeZ+EfJvvjVc2LrjBOLPFb/SO LNmV6m3l2gVkgwlEFOUhIfwL0WwyxxD81wLveoSASJkhqVYRckyD13ln Rg5EY+u7CCIXLdiUjewYaaw+Tgq9JtytTWYhCl0u/wcL/TyzYhcCDhHB DMI=

_psl.web.in.		43200	IN	TXT	"https://github.com/publicsuffix/list/pull/1089"
_psl.web.in.		43200	IN	RRSIG	TXT 8 3 86400 20200919101840 20200820081222 50998 web.in. URQLkIVUvEEi+V75Vhsu0Pp5O30zu+1AvGhTfGzjFUgKB7xtM5jfKpZ4 4y5aI7CzPSNGztEOkr+68LL+TYn38RZbk+GC7Jf1rfHOlyNLd2Kgaf34 oRC/ukh1er7bB2/9DrxmAFa9ki2prd6TIdlXJDu5uJzUcCVE64xFlBRA S/k=

_psl.gov.gd.		43200	IN	TXT	"https://github.com/publicsuffix/list/pull/1089"
_psl.gov.gd.		86400	IN	RRSIG	TXT 8 3 86400 20200919115144 20200820081309 26580 gov.gd. XRJM+5FX52139JKVsBVG2y1cqkEnFZ7jS5cdy3PVTHWGRcRm2YRq5wMd ha98Vu/R45gfTXLgEKPxNmiHLFCTkoNDVFYQ78ehmkord06Aymh/U5h6 hSejJB0dbskxY162r0K7WgZrk43GUWOwmNlAmz+H2WS9Xu6gXJaQWqFM +FA=

_psl.edu.gd.		43200	IN	TXT	"https://github.com/publicsuffix/list/pull/1089"
_psl.edu.gd.		43200	IN	RRSIG	TXT 8 3 86400 20200919163939 20200820081206 5713 edu.gd. ArTdLPA54Cw+0LO8wD4Is5h1qhaQnRlDWG3/poMo+5OKGr4tR3AavKgU l4D6pzDXv+XsJfEJSO0WhY/fRXaBtiTLDGUA1Xc90I9/fUgOK4qKY66j NkdYeQwN+mj5y4XIoy3B+jSdRfETWg1lEGrehddu8pmI44qkEV8+u6XG +nQ=

_psl.org.fm.		43200	IN	TXT	"https://github.com/publicsuffix/list/pull/1089"
_psl.org.fm.		86400	IN	RRSIG	TXT 8 3 86400 20200919145909 20200820081234 3346 org.fm. foRatDQvGdx2BK/61oKv14/IO1LHM0n0xUSiCL3eFr8W77/LIyqflFVo BeFjk2cQCvBJW64EuK/4sWKHuDT1e+XfpgLAVI3UFWRKLcDdxXvqwPdr Sx5DlPoQyiL5rIasdytMp8sueTIvggytw6gWdrDw1bkbfmb1zHYTYc4u Ie0=

_psl.net.fm.		43200	IN	TXT	"https://github.com/publicsuffix/list/pull/1089"
_psl.net.fm.		43200	IN	RRSIG	TXT 8 3 86400 20200919075813 20200820081231 9243 net.fm. QzK54pwtoNkLOEfeeExqPBunw1e858qiJ0+LJOcs5uGlkkFt2l8iZJXB ec2ZatXtP2DlEY9tDocQKDZQYGPdnSa3gnzKgSs0S09kXqku3+QSsBBn m8PD/fsUu5SiMlMh01WvO06DDeA1zVgASfFxWluZn8YZ4A6BvueEupSI Buo=

_psl.edu.fm.		43200	IN	TXT	"https://github.com/publicsuffix/list/pull/1089"
_psl.edu.fm.		43200	IN	RRSIG	TXT 8 3 86400 20200919131809 20200820081227 22909 edu.fm. GquwUHWOFm39mlL412mSrWYMIQc/vHiR+bJPYMTT3b/IgAx0t+xxwlAX DHip7HZxhGVxKEnO73y6YROxW52RQFCIp7Ia+VX4/9I2gD7QJKdj7gX3 jVIuJNN+BkeTir5RGRDY5Bzk352RdZWaar8DWw1OQ6jdAHu3kkVmi3V1 Iuc=

_psl.com.fm.		43200	IN	TXT	"https://github.com/publicsuffix/list/pull/1089"
_psl.com.fm.		86400	IN	RRSIG	TXT 8 3 86400 20200919024409 20200820081228 64366 com.fm. msDkbkYAiqxrKAlY0Dw/NrqN/Lwj9648R9RJL9g4RwbXeZKBHfvNkfb0 34s9xVXIFMbRwP2xH6mK38WNgSqj0q3TGG6AjHkzJHIx4v6ukwRx4WnK jzJ2fjnN470Lgl1HTI4VOAy4vsof7YaPlAxpaH0iLcbSztPGnIrJ8Iz2 khU=
```

make test
=========

I'm currently unable to run tests, without diving into a libicu/homebrew dependency rabbit hole, due to libpsl compile issues on macOS, but all tests up to and including `test_wildcard` passed successfully.

### CODE REVIEW

File: public_suffix_list.dat [hunk 1]
```diff
@@ -10957,6 +10967,21 @@ us.org
 // Submitted by Gavin Brown <gavin.brown@centralnic.com>
 co.com
 
+// Roar Domains LLC : https://roar.basketball/
```
Comment (dnsguru) [inline public_suffix_list.dat:10970]: DNS Validated

File: public_suffix_list.dat [hunk 2]
```diff
@@ -10957,6 +10967,21 @@ us.org
...
+// Submitted by Gavin Brown <gavin.brown@centralnic.com>
+aus.basketball
```
Comment (dnsguru) [inline public_suffix_list.dat:10972]: DNS Validated

File: public_suffix_list.dat [hunk 3]
```diff
@@ -1014,6 +1014,10 @@ pro.fj
 *.fk
 
 // fm : https://en.wikipedia.org/wiki/.fm
+com.fm
```
Comment (dnsguru) [inline public_suffix_list.dat:1017]: DNS Validated

File: public_suffix_list.dat [hunk 4]
```diff
@@ -1014,6 +1014,10 @@ pro.fj
...
+edu.fm
```
Comment (dnsguru) [inline public_suffix_list.dat:1018]: DNS Validated

File: public_suffix_list.dat [hunk 5]
```diff
@@ -1014,6 +1014,10 @@ pro.fj
...
+net.fm
```
Comment (dnsguru) [inline public_suffix_list.dat:1019]: DNS Validated

File: public_suffix_list.dat [hunk 6]
```diff
@@ -1014,6 +1014,10 @@ pro.fj
...
+org.fm
```
Comment (dnsguru) [inline public_suffix_list.dat:1020]: DNS Validated

File: public_suffix_list.dat [hunk 7]
```diff
@@ -1053,6 +1057,8 @@ ga
 gb
 
 // gd : https://en.wikipedia.org/wiki/.gd
+edu.gd
```
Comment (dnsguru) [inline public_suffix_list.dat:1060]: DNS Validated

File: public_suffix_list.dat [hunk 8]
```diff
@@ -1053,6 +1057,8 @@ ga
...
+gov.gd
```
Comment (dnsguru) [inline public_suffix_list.dat:1061]: DNS Validated

File: public_suffix_list.dat [hunk 9]
```diff
@@ -10957,6 +10967,21 @@ us.org
...
+
+// BRS Media : https://brsmedia.com/
+// Submitted by Gavin Brown <gavin.brown@centralnic.com>
+radio.am
+radio.fm
+
+// Globe Hosting SRL : https://www.globehosting.com/
+// Submitted by Gavin Brown <gavin.brown@centralnic.com>
+co.ro
+shop.ro
```
Comment (dnsguru) [inline public_suffix_list.dat:10983]: DNS Validated

File: public_suffix_list.dat [hunk 10]
```diff
@@ -10957,6 +10967,21 @@ us.org
...
+nz.basketball
+
+// BRS Media : https://brsmedia.com/
+// Submitted by Gavin Brown <gavin.brown@centralnic.com>
+radio.am
+radio.fm
+
+// Globe Hosting SRL : https://www.globehosting.com/
+// Submitted by Gavin Brown <gavin.brown@centralnic.com>
+co.ro
```
Comment (dnsguru) [inline public_suffix_list.dat:10982]: DNS Validated

File: public_suffix_list.dat [hunk 11]
```diff
@@ -10957,6 +10967,21 @@ us.org
...
 
+// Roar Domains LLC : https://roar.basketball/
+// Submitted by Gavin Brown <gavin.brown@centralnic.com>
+aus.basketball
+nz.basketball
+
+// BRS Media : https://brsmedia.com/
+// Submitted by Gavin Brown <gavin.brown@centralnic.com>
+radio.am
+radio.fm
```
Comment (dnsguru) [inline public_suffix_list.dat:10978]: DNS Validated

File: public_suffix_list.dat [hunk 12]
```diff
@@ -10957,6 +10967,21 @@ us.org
...
 co.com
 
+// Roar Domains LLC : https://roar.basketball/
+// Submitted by Gavin Brown <gavin.brown@centralnic.com>
+aus.basketball
+nz.basketball
+
+// BRS Media : https://brsmedia.com/
+// Submitted by Gavin Brown <gavin.brown@centralnic.com>
+radio.am
```
Comment (dnsguru) [inline public_suffix_list.dat:10977]: DNS Validated

File: public_suffix_list.dat [hunk 13]
```diff
@@ -10948,6 +10957,7 @@ gr.com
 // Radix FZC : http://domains.in.net
 // Submitted by Gavin Brown <gavin.brown@centralnic.com>
 in.net
+web.in
```
Comment (dnsguru) [inline public_suffix_list.dat:10960]: DNS Validated

**Comment by dnsguru** [comment]
@gbxyz This was very comprehensive as a patch - still under review for the names which are moving out from the Centralnic 3LD section.  Getting @weppos to give second set of eyeballs on this PR before merging given the number of affected TLDs, but tests ok and LGTM

**Comment by dnsguru** [comment]
@weppos (or @sleev) can you take a quick look at this to verify the approval and merge or ask questions?

**Comment by dnsguru** [comment]
@gbxyz just a clarification related to the strings that are moving out of the Centralnic section:
```
ar.com
gb.com
hu.com
kr.com
no.com
qc.com
uy.com
```
**For the benefit of the record here, can you confirm that these are not going to accept registrations or be treated as eTLD under Centralnic, but are under new administrative control?  Do you not, at least, operate DNS zone(s) for these any longer to where we could validate things with the TXT record?**

**My opinion** is that the approach you have taken to move the strings out to a separate section for these is the appropriate (and conservative) way to handle the change.    The preference would be to remove these, but due to the impact on existing _stuff_ the gaining party per name can request removal or section changes through validation of DNS _PSL TXT records.

**Comment by gbxyz** [comment]
> **For the benefit of the record here, can you confirm that these are not going to accept registrations or be treated as eTLD under CentralNic, but are under new administrative control? Do you not, at least, operate DNS zone(s) for these any longer to where we could validate things with the TXT record?**

This is confirmed. We removed these domains from our platform some time ago since the domains are now under the control of third parties, We don't host the DNS for them any more, so we can't publish a TXT records.

It may be that the new owners of these domains *might* decide to operate registries for some or all of them, or use them to host UGC in such a way as to justify their continued presence in the PSL. Or they may wish to remove the domains so that they work in the same way as "normal" domains. However, that's their decision to make.

**Comment by weppos** [comment]
@gbxyz any chance you can [enable the edit from maintainers](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) in this PR? There's a couple of sorting adjustments I'd like to make, and I can make them directly inline.

Beyond that, I have no objection. Just one note regarding the particolar point below, which is not necessarily a blocker to me to move on with the PR (after the sorting is addressed).

> My opinion is that the approach you have taken to move the strings out to a separate section for these is the appropriate (and conservative) way to handle the change. The preference would be to remove these, but due to the impact on existing stuff the gaining party per name can request removal or section changes through validation of DNS _PSL TXT records.

This is controversial. On one side, I understand the conservative approach and the desire to reduce unexpected changes. On the other side, I'm also not excited to keep potentially unnecessary rules forever. We've never really discussed removal of stale entries, but I would argue that an automated process should consider the lack of validation as a reason to remove.

A compromise could be a removal deadline: we reach out the owners as a courtesy, we give the new owner X days to confirm the entries, we remove them unless confirmed.

@Gavin out of curiosity and from your experience, do you expect the new owner will be able to follow up with us in a timely fashion?

**Comment by gbxyz** [comment]
> @gbxyz any chance you can enable the edit from maintainers in this PR? There's a couple of sorting adjustments I'd like to make, and I can make them directly inline.

@wepoos I'm afraid I can't. The "Allow edits from maintainers" checkbox isn't there.

> @gavin out of curiosity and from your experience, do you expect the new owner will be able to follow up with us in a timely fashion?

I'm afraid I don't know enough about the new owners to be sure.

**Comment by dnsguru** [comment]
> > My opinion is that the approach you have taken to move the strings out to a separate section for these is the appropriate (and conservative) way to handle the change. The preference would be to remove these, but due to the impact on existing stuff the gaining party per name can request removal or section changes through validation of DNS _PSL TXT records.
> 
> This is controversial. On one side, I understand the conservative approach and the desire to reduce unexpected changes. On the other side, I'm also not excited to keep potentially unnecessary rules forever. We've never really discussed removal of stale entries, but I would argue that an automated process should consider the lack of validation as a reason to remove.

As we do not have either a stale removal process or automation, I will proceed and set a deadline for the moved strings to PR and re-validate to remain listed, and proceed.  We should have a plan and policy for removing stale entries, but for resourcing this project via volunteers, pragmatically seeking to not invest disposable labor, that would still not happen fast.  

As far as deadlines go, we haven't means to communicate with the impacted entries' domain owners.   I'd like to proceed with the merge with the caveat that we'll remove these strings if they don't post TXT records reflecting THIS PR by October 15, 2020. 
 @gbxyz would you be able to communicate October 15, 2020 as a deadline to the party(ies) for them to open a PR and provide DNS TXT record validation?  One alternative path would be to require all these under your bailiwick to validate DNS entries, and remove those which don't.  

If that's fine, and I can get an acknowledgement from @gbxyz  to do that communication and an agreement from @weppos this is an acceptable path, I will merge and proceed, with an eye to address the egress strings later.  

Otherwise, it might be best if we were to split this comprehensive PR up, and move the contentious part to its own new PR so that the "Centralnic Section" is in its own, separate PR, so the other etld are not blocked.

**Comment by dnsguru** [comment]
@gbxyz in further review, we shall approve and merge, requesting that you notify the gaining party(ies) of the domains of the PSL entries being there, and the benefits and drawbacks, so that they can opt to file addendums to this PR or another.

**Comment by gbxyz** [comment]
> @gbxyz in further review, we shall approve and merge, requesting that you notify the gaining party(ies) of the domains of the PSL entries being there, and the benefits and drawbacks, so that they can opt to file addendums to this PR or another.

Will do.

**Comment by groundcat** [comment]
Hello @gbxyz , it seems that the project or organization associated with your domains, which was merged into the Public Suffix List (PSL), may no longer be active based on the observable usage of subdomains. 

```
hu.com
kr.com
uy.com
```

If the service has been decommissioned and the domain is no longer linked to an active project, as outlined in your original request, please confirm whether it can be safely removed from the PSL. This will help prevent leaving unnecessary entries for PSL volunteers and future registrants, as requesters often "set and forget" and neglect to request the removal of their domains from the PSL when their project reaches its end of life (EOL).

The PSL is a globally used resource, and it's important to understand that tens of millions, if not hundreds of millions, of devices and users incorporate the list. We aim to keep it as minimal as possible for those who truly need PSL inclusion. Projects that are smaller in scale, such as those without thousands of users or subdomains belonging to thousands of individual entities, will likely be ineligible for inclusion. If you no longer need your entry to remain on the PSL, **please create a new pull request to remove it** or reply here to confirm.

If your project or organization is still active and requires continued inclusion in the PSL, please reply at your earliest convenience. If we do not hear back from you for an extended period, **your entry may be subject to removal**. 

Additionally, ensure that the required `_psl` TXT record is maintained, as specified in the [PSL Guidelines](https://github.com/publicsuffix/list/wiki/Guidelines#validation-of-submissions--always), and that your domain is kept renewed with an expiry date of more than 2 years at all times to avoid future removal.

Thank you.

image

**Comment by gbxyz** [comment]
@groundcat as you will see from the conversation above, CentralNic sold those domains many years ago, and I left CentralNic last year in any case.

As discussed with @dnsguru back in 2020, I attempted to contact the owners of the domains but I don't recall ever getting an answer. If the PSL's policy now is to remove entries that lack a DNS proof, then by all means go ahead.

**Comment by groundcat** [comment]
Thank you @gbxyz , these entries have been removed. By the way, if you could ping someone at CentralNic to ask them to update the contact information (preferably with a department/team-based email address rather than a personal one), that would be appreciated. 

Currently, many CentralNic entries have `Submitted by Gavin Brown <gavin.brown@centralnic.com>`, but since you've left, this email address is no longer valid. 

In the future, when PSL maintainers want to contact someone at CentralNic, these email addresses in the comment lines are their first point of contact.

**Comment by groundcat** [comment]
Hello @gbxyz I have noticed that the domains `nz.basketball` and `aus.basketball` have been returning `NXDOMAIN` errors, and their WHOIS status shows as `inactive`. 

Could you please confirm if these projects are still active? If the services have been decommissioned and the domains are no longer linked to active projects similar to #2136, kindly confirm whether they can be safely removed from the PSL. If your organization still requires continued inclusion in the PSL, please reply at your earliest convenience. If we do not hear back from you for an extended period (over 30 days), your entries may be subject to removal. Thank you. #1119

**Comment by gbxyz** [comment]
Apologies, but I can't help you, you will need to contact the registry operator themselves.

**Pull request merged**